stubs/croniter/METADATA.toml

```diff
@@ -1,2 +1,2 @@
-version = "0.1"
-python2 = true
+version = "1.0"
+python2 = false
```

Comment:
According to croniter's `setup.py`, Python 2 is still supported. Any reason to drop it here?